pkg/sql/pgwire/testdata/pgtest/prepare

```diff
@@ -92,13 +91,31 @@ ReadyForQuery
 {"Type":"ReadyForQuery","TxStatus":"I"}
 
 send
-Parse {"Query": "EXPLAIN ANALYZE SELECT 1;"}
+Parse {"Name": "e", "Query": "EXPLAIN SELECT generate_series(1,10);"}
+Bind {"DestinationPortal": "pe", "PreparedStatement": "e"}
+Execute {"Portal": "pe"}
 Sync
 ----
 
-until crdb_only keepErrMessage
-ErrorResponse
+until ignore=DataRow
+ReadyForQuery
+----
+{"Type":"ParseComplete"}
```

Comment:
i think a test case should be added to TestPrepareExplain in pkg/sql/explain_test.go. actually if you so that, we may not even need the testdata/pgtest ones, unless they actually are testing something more.